style.css

```diff
@@ -0,0 +1,318 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;600;700&display=swap');
+
+h1 {
+  font-size: 40px;
+  color: #172b4d;
+  font-weight: 700;
+  max-width: 300px;
+  margin-bottom: 16px;
+  margin-top: 68px;
+  line-height: 52px;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  color: #172b4d;
+  margin-top: 0;
+  position: absolute;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 0 auto;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+}
+
+.logo {
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  color: #6070ff;
+  position: relative;
+  text-decoration: none;
+  margin-top: 30px;
+}
+
+h4 {
+  font-weight: 700;
+  line-height: 15px;
+  letter-spacing: 0;
+  color: #7f8cff;
+}
+
+ul {
+  display: inline-block;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.content {
+  background-image: url(img/header-shapes-mobile.png);
+  background-repeat: no-repeat;
+  height: 100vh;
+}
+
+.subcontent {
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+ul li {
+  margin-right: 10px;
+}
+
+.Union {
+  position: absolute;
+  left: 322px;
+  top: 27px;
```

Comment:
as mentioned above you don't need absolute position as this will put this item out of the dom flow  while we want it to be responsive  I suggest that you remove this attributes and the flexbox container will align your items correctly 😊

Reply:
done